### PR TITLE
include go in the project

### PR DESCRIPTION
**Pull request by phoracek**
In order not to get different results on different systems, let's include
specific Go binary under the project with custom GOROOT.

Signed-off-by: Petr Horacek <phoracek@redhat.com>

### CODE REVIEW

**Comment by phoracek** [comment]
cc @RamLavi @qinqon

File: Makefile [hunk 1]
```diff
@@ -1,20 +1,31 @@
...
+
+BIN_DIR = $(CURDIR)/build/_output/bin/
+
 export GOFLAGS=-mod=vendor
 export GO111MODULE=on
-
-IMG ?= kubevirt/kubemacpool
+export GOROOT=$(BIN_DIR)/go/
+export GOBIN=$(GOROOT)/bin/
+export PATH := $(GOROOT)/bin:$(PATH)
```
Comment (qinqon) [inline Makefile]: We can use GOBIN directly here.

**Comment by qinqon** [comment]
@phoracek found an alternative way to do this using "go get" https://github.com/qinqon/kube-admission-webhook/blob/master/hack/install-go.sh

**Comment by phoracek** [comment]
@qinqon are you using `go` in order to install `go`? :D Doesn't it defy the purpose?

**Comment by phoracek** [comment]
This fails only on the one problematic test. I don't believe it is a regression. @RamLavi what do you think?

**Pull request merged**